reference/under_the_hood/indexation.md

```diff
@@ -14,10 +14,54 @@ Crashes caused by memory overconsumption can still happen in two cases:
+
+- **Bigger HTTP payloads are processed more quickly than smaller payloads**. For example, adding the same 100,000 documents in two batches of 50,000 documents will be quicker than in four batches of 25,000 documents. By default, MeiliSearch sets the maximum payload size to 100MB, but [you can change this value if necessary](/reference/features/configuration.md#payload-limit-size). That said, **the bigger the payload, the higher the memory consumption**. An instance may crash if it requires more RAM than is currently available in a machine.
+
+- **MeiliSearch should not be your main database**. It means you should only insert the documents you want to retrieve using the search, and no more. The more you add documents, the more the indexation and search time will increase.
+
+- By default, all the fields of your documents are considered as "searchable". We strongly recommend changing this by [setting the `searchablaAttributes`](https://docs.meilisearch.com/reference/api/searchable_attributes.html#update-searchable-attributes) with the exhaustive list of fields you want to search in. The less there are fields to index, the more the indexation time will decrease.
+For example, with the following documents, you only need to put `title` and `author` as `searchableAttributes`, because you might not want to perform the search in `id` or `genre` fields.
+
+```json
+[
+  { "id": 1, "title": "Pride and Prejudice", "author": "Jane Austin", "genre": "romance" },
+  { "id": 2, "title": "Le Petit Prince", "author": "Antoine de Saint-Exupéry", "genre": "adventure" }
+]
+```
+
+- The first time you push your documents, we recommend pushing your settings first, and then, add your documents. Not the contrary. It will drastically decrease the time of indexation. `searchableAttributes` described in the previous point is part of these settings.
+
+- Since indexation speed is tightly connected to the size of your payload, using lightweight dataset formats such as CSV and NDJSON can lead to increased performance. Be aware converting your JSON dataset into CSV will not work for array fields.
+
+- Do not run MeiliSearch in machine using a HDD (Hard Disk Drive). Most of the recent cloud providers do not offer this kind of service anymore, but it can still happen.
+
+- Ensure your cloud provider does not limit your I/O operations, like AWS does with their [Amazon EBS service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html) for example.
+
+## Memory crashes
+
+In some cases, MeiliSearch can be stopped by the OS. Most of these crashes happen during indexation and are a result of a machine running out of RAM. This happens when your computer does not have enough memory to process your dataset.
+
+Additionally, indexation uses disk space. MeiliSearch may also crash if it runs out of disk space while indexing new documents.
```

Comment:
```suggestion
Additionally, indexation uses disk space. MeiliSearch can throw an error if it runs out of disk space while indexing new documents.
```

Reply:
Since it's not a crash issue anymore, I remove the complete sentence